### PR TITLE
Update structure.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**

### CODE REVIEW

File: source/menus/structure.html.erb.md [hunk 1]
```diff
@@ -30,8 +32,8 @@ In the following example, the user is not required to read the home page before
 <ul>
 	<li><a href="…">Home</a></li>
 	<li><a href="…">Shop</a></li>
-	<li><a href="…">SpaceBears</a></li>
-	<li><a href="…">MarsCars</a></li>
+	<li><a href="…">Space Bears</a></li>
```
Comment (nitedog) [inline source/menus/structure.html.erb.md]: Really minor but I thought separation of words makes it easier to ready for screen reader users.

Comment (yatil) [inline source/menus/structure.html.erb.md]: The idea was to have this look as a trademark name. I am not feeling strongly about this at all.

File: source/menus/structure.html.erb.md [hunk 2]
```diff
@@ -10,17 +10,19 @@ wcag_techniques:
   - ARIA11
 ---
 
-Semantic markup conveys the menu structure to users. It makes the menu adaptable to different situations, such as small screen display, zoomed in, or assistive technology use.
+Semantic markup conveys the menu structure to users. It makes the menu adaptable to different situations, such as small screen displays, magnification, and use with assistive technology. Styling is applied later to give this structure the desired visual appearance of the menu and its items.
```
Comment (yatil) [inline source/menus/structure.html.erb.md]: - It says “small screen display” because it is displayed on a small screen. I _think_ we need to say “It makes the menu more adaptable to different situations, such as being shown on small screen displays, with magnifications, and used with assistive technologies.” I think I try to simplify the wording here, maybe splitting into two sentences makes more sense.
- I don’t think we need the sentence “Styling is applied later to give this structure the desired visual appearance of the menu and its items.” – We make that pretty clear in the index and by not talking about styling here.

Comment (yatil) [inline source/menus/structure.html.erb.md]: What about “Menus can adapt to different situations, such as small screen displays, magnification, and use with assistive technology.” or even “Menus can adapt to different situations, such as small screen displays, magnification, and other assistive technology.”

File: source/menus/structure.html.erb.md [hunk 3]
```diff
@@ -10,17 +10,19 @@ wcag_techniques:
...
 
-## Menus as List of Links
+## Menu representation
```
Comment (yatil) [inline source/menus/structure.html.erb.md]: I am not a fan of abstract wording, but I am also not feeling strongly.

File: source/menus/structure.html.erb.md [hunk 4]
```diff
@@ -68,21 +70,9 @@ In instances where pages need to be read in a certain sequence, for example chap
...
-{::nomarkdown}
-<%= ref :middle %>
-{:/}
-
-For more information on regions, see the [page regions](/page-structure/regions.html) tutorial.
-
-{::nomarkdown}
-<%= ref :end %>
-{:/}
+The following example uses the HTML5 `<nav>` element to identify a menu. Other techniques are described in the [page regions](/page-structure/regions.html) tutorial.
```
Comment (yatil) [inline source/menus/structure.html.erb.md]: I think we explicitly said to not use inline-links in the carousels tutorial. Why change that approach here?

File: source/menus/structure.html.erb.md [hunk 5]
```diff
@@ -179,7 +143,9 @@ The item has also an invisible text “Current Page:” added to the menu item.
 ### Using WAI-ARIA
 {:.ap}
 
-If a link to the main content area of the page is required or if the `<a>` element cannot be removed from the navigation, use `aria-current="page"` as an alternative.
+Use the `aria-current="page"` attribute to indicate the current menu item and its type. This is particularly useful for application menus and ordered menus, such as instructional steps.
+
+In the following example WAI-ARIA is used to indicate the current menu item:
```
Comment (yatil) [inline source/menus/structure.html.erb.md]: I think we can live without that last sentence.

File: source/menus/structure.html.erb.md [hunk 6]
```diff
@@ -179,7 +143,9 @@ The item has also an invisible text “Current Page:” added to the menu item.
 ### Using WAI-ARIA
 {:.ap}
 
-If a link to the main content area of the page is required or if the `<a>` element cannot be removed from the navigation, use `aria-current="page"` as an alternative.
+Use the `aria-current="page"` attribute to indicate the current menu item and its type. This is particularly useful for application menus and ordered menus, such as instructional steps.
```
Comment (yatil) [inline source/menus/structure.html.erb.md]: Quick comment:

> This is particularly useful for application menus and ordered menus, such as instructional steps.

I don’t think that is correct: Application menus have no state (they execute an action), and for instructional steps it would be better to use `aria-current="step"`…

I’ll provide a rewording in a pull request.

File: source/menus/structure.html.erb.md (hunk 2 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: I like the direction but not all menus can adapt - only menus with appropriate markup. How about something like "Menus with good structure can adapt..."?

File: source/menus/structure.html.erb.md (hunk 3 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: Me too. Maybe "menu body" or such?

File: source/menus/structure.html.erb.md [hunk 7]
```diff
@@ -10,17 +10,19 @@ wcag_techniques:
...
 {:#lists-of-links.newap}
 
-Convey the menu structure to users effectively by using a list, especially if the menu consists of more than very few links. With this structural information assistive technologies can announce the number of items in the menu.
+Convey the menu structure, typically by using a list. Such structural information allows assistive technologies to announce the number of items in the menu and provide corresponding navigation functionality.
 
 ### Unordered list
 {:.ap}
 
-In the following example, the user is not required to read the home page before advancing to the shop or get information about the products. As the sequence of reading the pages is not important to understand the website as a whole, use an unordered list (`<ul>`) – this approach can be used on most websites.
+Use an unordered list (`<ul>`) when the menu items do not represent a particular sequence. This is usually the case for most types of menus, such as website navigation.
```
Comment (nitedog) [inline source/menus/structure.html.erb.md]: I think the mixing of factual information and example is confusing. I tried to separate them by providing the fact first, and in the next paragraph the example to make it concrete. Happy to get other perspectives as needed.

File: source/menus/structure.html.erb.md [hunk 8]
```diff
@@ -10,17 +10,19 @@ wcag_techniques:
...
+
+In the following example, the sequence of the menu items is not necessarily the intended order for accessing these pages of a website:
```
Comment (nitedog) [inline source/menus/structure.html.erb.md]: This should be an instance of the general principle in the previous paragraph. Maybe we can make it more concrete with something like:

"In the following example, the home page menu item does not need to be accessed before the product information item"

File: source/menus/structure.html.erb.md (hunk 1 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: Me too. Your call...

File: source/menus/structure.html.erb.md (hunk 4 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: I don't feel strongly - feel free to change as needed.

The rationale is that the link is clear and optional from the context. It is not like linking to the definition of a word or such, where the reader is unsure if they need to follow the link or not.

File: source/menus/structure.html.erb.md (hunk 6 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: OK, agreed.

File: source/menus/structure.html.erb.md (hunk 5 above)
Comment (nitedog) [inline source/menus/structure.html.erb.md]: OK.

File: source/menus/structure.html.erb.md (hunk 8 above)
Comment (yatil) [inline source/menus/structure.html.erb.md]: I wonder if you have seen my PR #433 below? If you’re ok with the changes there, I’m happy to merge that into this PR and then the whole thing into the current version :-)
